data/ga/legislature/Richard-Smith-d0827b60-7e78-46e8-944a-08e292704805.yml

```diff
@@ -0,0 +1,38 @@
+id: ocd-person/d0827b60-7e78-46e8-944a-08e292704805
+name: Richard Smith
+given_name: Richard
+family_name: Smith
+email: richard.smith@house.ga.gov
+image: https://www.legis.ga.gov/api/images/default-source/portraits/smith-richard-213.jpg
+party:
+- name: Republican
```

Comment:
@sroomf this person was in Georgia House district 134 from January 2015 until 01/09/2023 and was in district 131 from January 2005 until January 2015.